yarn.lock

```diff
@@ -865,6 +1008,24 @@ heroku-client@^2.4.0:
     q "^1.2.0"
     tunnel-agent "^0.4.0"
 
+heroku-container-registry@4.3.0:
+  version "4.3.0"
+  resolved "http://cli-npm.heroku.com/heroku-container-registry/-/heroku-container-registry-4.3.0/433e6e311ac8c289b3f1845e9b1f633483f3c390.tgz#433e6e311ac8c289b3f1845e9b1f633483f3c390"
+  dependencies:
+    camelcase "1.0.2"
```

Comment:
the only dependencies here that I see used are glob, heroku-cli-util, and inquirer. The others all don't seem to be referenced at all.